catalogue/webapp/test/e2e/selectors/common.ts

```diff
@@ -0,0 +1,21 @@
+export async function fillInput(id: string, text: string): Promise<void> {
+  await page.fill(id, text);
```

Comment:
What do we gain through these abstractions? 

To nit as well - are these selectors? Or actions?

Reply:
the advantage is mainly when we need to maintain or update the api. We only need to change it in one place if the api changes but can change this.

Yeah good point on selectors and actions. Renaming  functions or separation. Renaming functions for me is better for now. Thoughts?